src/collective/taxonomy/indexer.py

```diff
@@ -49,8 +49,9 @@ def __call__(self):
                 if identifier in stored_element:
                     found.append((identifier, language, path,))
 
+        lang = getattr(self.context, 'language', language)
```

Comment:
I don't understand why is it not ok to acquire language on context ? 
Maybe we could get default portal language if context has no language ?